src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -20,21 +16,33 @@
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+                new Telegram("alexyeoh"),
```

Comment:
A minor thing, but perhaps the handle has to start with @ right?

Reply:
For input I think we can leave it as that, without the prefix `@`.

I have updated the `toString` instance to reflect the handle.